test/sig_utils.py

```diff
@@ -67,6 +72,8 @@ def get_cpp_type(stan_type):
     "categorical_rng": [simplex, None],
     "categorical_lpmf": [None, simplex],
     "cholesky_decompose": [pos_definite, None],
+    "csr_to_dense_matrix": [1, 1, None, [1], [1, 2]],
+    "csr_matrix_times_vector": [1, 1, None, [1], [1, 2], None],
```

Comment:
It was a workaround for the if statement [here](https://github.com/stan-dev/math/blob/develop/test/expressions/generateExpressionTests.py#L66).

Now how these magic constants work is if there is an array argument, you can type in an array here and it will translate to a cpp array initializer like:

```
std::vector<double> myvec = { 1, 2 };
```

It doesn't remove the tricky logic, but it moves it to the same place as a lot of the other tricks at least.

Reply:
That is nice!